src/styles/global.scss

```diff
@@ -138,6 +140,12 @@ body {
 .catch-phrase{
   font-family: Zilla Slab, san-serif, serif;
   font-style: italic;
-  color: white;
+  color: white
```

Comment:
Why did you remove the semi-colon? While you can technically do this, it can lead to errors. I'd restore it, unless there's a good reason 😄

Reply:
Done 😺